shale/src/lib.rs

```diff
@@ -252,7 +227,7 @@ pub trait ShaleStore<T: Send + Sync> {
 /// A stored item type that can be decoded from or encoded to on-disk raw bytes. An efficient
 /// implementation could be directly transmuting to/from a POD struct. But sometimes necessary
 /// compression/decompression is needed to reduce disk I/O and facilitate faster in-memory access.
-pub trait Storable {
+pub trait Storable: Debug + Send + Sync {
```

Comment:
Why put the bounds here? Is there anything in the trait that actually requires `Debug`, `Send`, or `Sync`? 

```suggestion
pub trait Storable {
```

I checked clippy this time too and everything passed without this.